server/start.go

```diff
@@ -193,6 +193,25 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.AppCreator) error {
 	cfg := ctx.Config
 	home := cfg.RootDir
+	var cpuProfileCleanup func()
+
+	if cpuProfile := ctx.Viper.GetString(flagCPUProfile); cpuProfile != "" {
```

Comment:
Good one :+1: 

Why does `flagCPUProfile` accepts a string? Shouldn't just be a boolean flag? I'd understand if this was reckoned to be out of this PR's scope though, thus I'm not asking to change this (at least not necessarily in this PR).